### DISCUSSION TITLE
How do I have light mode components?

### DISCUSSION THREAD

**Discussion by NeedNot** [unovue/shadcn-vue#615] [Q&A] [answered]
All the components are dark but i want to use light theme

**Comment by Saeid-Za** [comment, marked as answer]
Components use tailwind clasess to support both light and dark modes.
If you only see the dark mode, it's because of the "dark" class on body.
Any chance you've added that class to your body element, but you didn't mean to?
I'd suggest reading the themeing section in the docs.
https://www.shadcn-vue.com/docs/dark-mode.html

**Reply by NeedNot** [reply]
> Components use tailwind clasess to support both light and dark modes. If you only see the dark mode, it's because of the "dark" class on body. Any chance you've added that class to your body element, but you didn't mean to? I'd suggest reading the themeing section in the docs. https://www.shadcn-vue.com/docs/dark-mode.html

I don't but if i switch to css variables then it does default to light theme and putting dark does make it go to dark theme. i was just wanting to know if i could use light theme without css variables

**Reply by sadeghbarati** [reply]
You can force light mode. Just add `light` class to the `html` tag and remove dark classes
